mlfinlab/codependence/correlation.py

```diff
@@ -111,3 +111,51 @@ def distance_correlation(x: np.array, y: np.array) -> float:
     coef = np.sqrt(d_cov_xy) / np.sqrt(np.sqrt(d_cov_xx) * np.sqrt(d_cov_yy))
 
     return coef
+
+
+def kl_dist(corr_A: np.array, corr_B: np.array) -> float:
```

Comment:
Bad function name. Should this be a public or a hidden function?

Reply:
@skyliquid22 Let's switch this to `kullback_leibler_distance`